client/components/editor.vue

```diff
@@ -195,15 +239,36 @@
           return;
         }
 
+        const cursors = Cursor.documents.find(_.extend(this.cursorsHandle.scopeQuery(), {
+          clientId: {
+            $ne: this.clientId,
+          },
+        })).fetch();
```

Comment:
There is also a map function already available on Minimongo cursors. So you might want to use that instead of first fetching and then mapping.

Reply:
BTW, are you sure you want this inside this autorun? Why don't you make your own autorun for cursor managing purpose? Because now this will rerun as a whole every time a cursor or a step changes.